well-known/chains.go

```diff
@@ -0,0 +1,297 @@
+package wellknown
+
+import (
+	pbfirehose "github.com/streamingfast/pbgo/sf/firehose/v2"
+)
+
+type WellKnownProtocol struct {
+	Name          string
+	BlockType     string
+	BufBuildURL   string
+	BytesEncoding pbfirehose.InfoResponse_BlockIdEncoding
+	KnownChains   []*Chain
+}
+
+type Chain struct {
+	Name    string
+	Aliases []string
+	// Genesis block here is actually the "lowest possible" first streamable block through firehose blocks.
+	// In most cases, it matches the "genesis block" of the chain.
+	GenesisBlockID     string
```

Comment:
Document expected format, which I think it's chain specific. Probably good to document how to retrieve it too, so PR are easy to submit.